handler.js

```diff
@@ -1,20 +1,24 @@
+// @ts-check
 'use strict';
 
 const uuidv4 = require('uuid/v4');
 const request = require('request');
+// @ts-ignore aws-sdk is already installed on AWS. Not installing locally to keep the buidl artifacts small
 const AWS = require('aws-sdk');
 const Handlebars = require('handlebars');
 const he = require('he');
 const qs = require('qs');
+const slackHelper = require('./slack-helper');
 
 const ses = new AWS.SES();
 
 const EMAIL_TEMPLATE = Handlebars.compile(process.env.EMAIL_TEMPLATE,
-                                          {noEscape: true});
+  {noEscape: true});
+const EMAIL_SEPARATOR = ', ';
```

Comment:
The original code was a bit more liberal - accepting a comma or semicolon, then stripping the extra whitespace. Might be worth keeping it (or something similar) to make it easier for people to specify recipient lists?

Reply:
Good call, will do.

Reply:
This separator is only used internally by `ahv-council-thing`. The list of email addresses is first sent to the `submit` function as an array of strings – then the `submit` function joins those into a string for Slack, and the `approve` function needs to split them again.

Given that this is entirely internal to `ahv-council-thing` I think we're OK with a simple approach, let me know if you disagree.

Reply:
Ah yes, that makes sense. I missed the fact that it was in `approve`, not `submit`.